templates/paloalto_panos_show_arp_all.template

```diff
@@ -1,6 +1,6 @@
 Value INTERFACE (\S+)
 Value IP (\S+)
-Value MAC ([0-9a-fA-F:]+)
+Value MAC (\S+)
```

Comment:
I would prefer not to have the parenthesis captured. I think `(\(?\S+\)?)` should work